tests/browser/test/test_setup.js

```diff
@@ -306,21 +307,20 @@ async function connect(
   targetBlock,
   targetConnection,
   mutatorBlockId,
-  dragBlockSelector,
 ) {
   let draggedLocation;
   let targetLocation;
 
   if (mutatorBlockId) {
     draggedLocation = await getLocationOfBlockConnection(
       browser,
-      draggedBlock,
+      draggedBlock.id,
       draggedConnection,
       mutatorBlockId,
     );
     targetLocation = await getLocationOfBlockConnection(
       browser,
-      targetBlock,
+      targetBlock.id,
       targetConnection,
       mutatorBlockId,
     );
```

Comment:
I think we can remove this else and just always pass `mutatorBlockId` (which may be undefined).